insights-notification-schemas-java/src/main/java/com/redhat/cloud/notifications/validator/LocalDateTimeValidator.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.cloud.notifications.validator;
+
+import com.networknt.schema.Format;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class LocalDateTimeValidator implements Format {
```

Comment:
Is it tested?

Reply:
Indirectly tested here: https://github.com/RedHatInsights/insights-schemas-java/pull/37/files#diff-0646229727cc98872b02c02f67eaf6d8672f4ffbe4342fe92af15ceced587911R187-R212

Reply:
Adding some tests